src/hooks/useMinCollapsedLines.js

```diff
@@ -0,0 +1,19 @@
+import {useMemo} from 'react';
+import {expandCollapsedBlockBy} from '../utils';
+
+export default (minLinesExclusive, hunks, oldSource) => {
+    if (!oldSource) {
+        return hunks;
```

Comment:
On the first version I was throwing an exception when `oldSource` is not provided, but this may cause some issues:

1. Frequently when `oldSource` is not fetched via some HTTP requests, users want every feature to work smoothly
2. If we require them to "choose which hooks to use" based on the existence of `oldSource`, this causes different hook invocation orders on re-renders, which will break react fiber's assumption

As a result, all hooks deal the situation when `oldSource` is not provided

Reply:
OK...
Makes it heavy，but no problem to me.